bindata/oauth-apiserver/deploy.yaml

```diff
@@ -101,18 +101,36 @@ spec:
         - mountPath: /var/log/oauth-apiserver
           name: audit-dir
         livenessProbe:
-          initialDelaySeconds: 30
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
+          initialDelaySeconds: 0
+          periodSeconds: 10
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 3
```

Comment:
just confirm, these are the defaults as per https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes.

What is the rationale of going from 30s initial delay to 0?

Reply:
> What is the rationale of going from 30s initial delay to 0?

We're using a startup probe now, which allows for a different set of settings before passing the startup, then using a tight value after startup is complete.

Reply:
> just confirm, these are the defaults as per https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes.

Many are.  Abu thought it was easier to reason about with all the values listed.  I think he was right.